nixos/modules/config/console.nix

```
@@ -166,7 +166,7 @@ in
           # Add everything if we want full console setup...
           "/etc/kbd" = lib.mkIf cfg.earlySetup { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share"; };
           # ...but only the keymaps if we don't
-          "/etc/kbd/keymaps" = lib.mkIf (!cfg.earlySetup) { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share/keymaps"; };
+          "/etc/kbd/keymaps" = { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share/keymaps"; };
```

```suggestion
          "/etc/kbd/keymaps" = lib.mkIf (!cfg.earlySetup) { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share/keymaps"; };
          "/etc/kbd/consolefonts" = lib.mkIf (!cfg.earlySetup && cfg.font != null) { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share/consolefonts"; };
```

The important part here is that `/etc/kbd/consolefonts` is currently not being included in the initrd.

See also: https://github.com/NixOS/nixpkgs/issues/257904#issuecomment-2024304441